run_designs.py

```diff
@@ -32,7 +32,7 @@
 @click.option("-c", "--config_tag", default="config", help="Configuration file")
 @click.option("-r", "--regression", default=None, help="Regression file")
 @click.option("-t", "--tag", default="regression", help="Tag for the log file")
-@click.option("-j", "--threads", help="Number of designs in parallel")
+@click.option("-j", "--threads", default=4, help="Number of designs in parallel")
```

Comment:
Why?

Reply:
If -j is not passed, then run designs errors out cryptic message stating that empty string can't be converted to integer in line 118. Maybe should just set it to one.

Reply:
That's more reasonable, yeah.